### PR TITLE
Fix/boost asio io context

### PR DESCRIPTION
**Pull request by florianschroedter**
I am using Crow (single include) in a private project on Ubuntu. When trying to compile (g++ 9.4.0-1) with my other C++ 17 code, I got a compiler error in the socket_adaptors.h file, specifically the boost::asio::io_service& get_io_service() function, which I corrected with a cast to boost::asio::io_context&. Furthermore in I fixed a fallthrough warning in websocket.h.

This is my first open source contribution, I am not a professional or even advanced developer. I hope this helps and I did everything right on the pull request (never done this before either).

### CODE REVIEW

File: include/crow/websocket.h [hunk 1]
```diff
@@ -390,6 +390,7 @@ namespace crow
                                         message_handler_(*this, message_, is_binary_);
                                     message_.clear();
                                 }
+                                break; // Fixing fallthrough error
```
Comment (keith-dev) [inline include/crow/websocket.h:393]: Do we need the '// fixing' comment to be permaently in the cide?

File: include/crow/socket_adaptors.h [hunk 2]
```diff
@@ -19,7 +19,8 @@ namespace crow
 
         boost::asio::io_service& get_io_service()
         {
-            return socket_.get_io_service();
+            return static_cast<boost::asio::io_context&>(socket_.get_executor().context()); // Fix compilation error
```
Comment (keith-dev) [inline include/crow/socket_adaptors.h:22]: We probably shouldn't keep these new comments permanently.

**Comment by florianschroedter** [comment]
I have removed the comments.

**Comment by gittiver** [comment]
needs rebase

**Comment by mrozigor** [comment]
Obsolete PR.

**Pull request closed without merging**